querybook/config/user_setting.yaml

```diff
@@ -20,6 +20,15 @@ default_query_engine:
     options: []
     helper: Default query engine when creating a query cell.
 
+use_previous_query_engine:
+    per_env: true
```

Comment:
I set it per environment since the `default_query_engine` option is also set per environment I would guess it's good to have that option. But I can see how it's ok having it as a general setting as well.
Do you prefer it to be general?

Reply:
Sounds good 👍